### PR TITLE
Milestone 9: Add Devise

### PR DESCRIPTION
**Pull request by topeogunleye**
Milestone 9: Add Devise

> * [Modified the views in devise](https://github.com/heartcombo/devise#configuring-views) for registration and login to match [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) and your styling.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,8 @@ ruby '3.1.2'
 
 gem 'rubocop', '>= 1.0', '< 2.0'
 
+gem 'cancancan'
```
Comment (rustam-tolipov) [inline Gemfile:8]: Well, you made great effort for making this project. However, there are some errors that need to be fixed.

- [ ] Kindly make sure when you `Sign out` your page need to redirect you to login page.

File: app/views/posts/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Editing an Article</h1>
```
Comment (rustam-tolipov) [inline app/views/posts/edit.html.erb:1]: - [ ] When I tried to `reset` my password it's breaking the app. Based on the requirement, user is able to reset their password. Also enable in user model `:confirmable` for device.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @topeogunleye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Gemfile (hunk 1 above)
Comment (topeogunleye) [reply Gemfile:8]: @Rustamxon7, please do you have any articles or resource that I can use to learn how to redirect to homepage with sign out button

Comment (rustam-tolipov) [reply Gemfile:8]: I think that might be helpful for you. [Link...](https://github.com/heartcombo/devise/issues/5458#:~:text=You%20have%20to,data%3A%20%7B%20turbo_method%3A%20%3Adelete%22%20%7D%20%25%3E)

Comment (topeogunleye) [reply Gemfile:8]: @Rustamxon7, my sign out works but I have to reload my page to redirect to the homepage. please can you give some suggestions on how to fix this

**Review by brytebee — APPROVED** [review]
Hi, @topeogunleye👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great implementation of previous reviewer's suggestions.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**